ui/app/AppLayouts/Wallet/popups/AddCustomTokenModal.qml

```diff
@@ -148,6 +148,13 @@ ModalPopup {
         height: addBtn.height
         visible: editable
 
+        MessageDialog {
+            id: changeError
+            title: "Changing settings failed"
```

Comment:
```suggestion
            title: qsTr("Changing settings failed")
```

Reply:
done, plus added in `AccountSettingsModal` as well.